src/codeu/chat/codeU_db/CreateDatabase.java

```diff
@@ -0,0 +1,98 @@
+package codeu.chat.codeU_db;
+
+import java.sql.*;
+
+public class CreateDatabase
+{
+    public static void main( String args[] )
+    {
+        Connection c = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            c = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
+            System.out.println("Opened database successfully");
+        } catch ( Exception e ) {
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+            System.exit(0);
+        }
+
+        try {
+            stmt = c.createStatement();
+            String sql = "CREATE TABLE USERS " +
+
+                    "(ID            VARCHAR(16) PRIMARY KEY NOT NULL," +
+                    " UNAME         CHAR(25)    UNIQUE      NOT NULL, " +
+                    " TimeCreated   BIGINT               NOT NULL, " +
+                    " PASSWORD      TEXT                    NOT NULL)";
+            stmt.executeUpdate(sql);
+            stmt.close();
+        } catch ( Exception e ) {
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+            System.exit(0);
+        }
+        System.out.println("Table <USERS> created successfully");
+
+        try {
+            stmt = c.createStatement();
+            String sql = "CREATE TABLE CONVERSATIONS " +
```

Comment:
Looks like a lot of copy and pasted code in here. Perhaps create a helper function that does all the common stuff and just pass the connection and SQL string into that.